assets/pages/contributions-landing/containers/contributeContainer.js

```diff
@@ -0,0 +1,26 @@
+// @flow
+
+// ----- Imports ----- //
+
+import { connect } from 'react-redux';
+
+import Contribute from 'components/contribute/contribute';
+import { getPaymentLogosTestVariant } from 'helpers/abTests/helpers';
+import type { State } from '../contributionsLandingReducer';
+
+
+// ----- State Maps ----- //
+
+function mapStateToProps(state: State) {
+
+  return {
+    contributionType: state.page.selection.contributionType,
+    paymentLogosVariant: getPaymentLogosTestVariant(state.common.abParticipations),
```

Comment:
This is the only place where we are `getPaymentLogosTestVariant` is it worth having this function? Why don't we access here directly to the abParticipations object?

Reply:
Sorry we are also using it [here](https://github.com/guardian/support-frontend/pull/648/files#diff-1baefd1cc3cb6a599c98babede97179dR18) but maybe we can delete and access directly the object there too.

Reply:
It is also used here: https://github.com/guardian/support-frontend/blob/74ccf0d406f564500e63c692136bd8bdbcf75022/assets/helpers/abTests/helpers.js#L14

I think it's worth keeping because otherwise you have to hardcode the test name string in 3 places.